### ISSUE TITLE
Grafana service broken in master

### ISSUE THREAD

**Issue by bjornfor** [NixOS/nixpkgs#13718]
## Basic info

* Kernel: Linux talisa 4.3.6 #1-NixOS SMP Fri Feb 19 22:35:24 UTC 2016 x86_64 GNU/Linux
* System: 16.09pre78286.0bf8a1a (Emu)
* Nix version: nix-env (Nix) 1.11.2
* Nixpkgs version: "16.09pre78286.0bf8a1a"

## Describe your issue here
Grafana (```services.grafana.enable = true```) doesn't work anymore. It worked fine in 16.03pre77568.40c586b, but as of 16.09pre78286.0bf8a1a (Emu) it fails to start:

```
$ systemctl status grafana
● grafana.service - Grafana Service Daemon
   Loaded: loaded (/nix/store/szvqf6x8fmis1r4vc4j0pqrpgdhnljzh-unit-grafana.service/grafana.service; bad; vendor preset: enabled)
   Active: failed (Result: exit-code) since Sun 2016-03-06 19:57:32 CET; 20min ago
  Process: 666 ExecStart=/nix/store/xr947jw82kjjx9xgqq2imybkirdwy2vq-go1.5-grafana-v2.6.0-bin/bin/grafana -homepath /var/lib/grafana (code=exited, status=1/FAILURE)
  Process: 587 ExecStartPre=/nix/store/dvk2jg3g94g9vkwqzg7bhcn6fbvgilx9-unit-script/bin/grafana-pre-start (code=exited, status=0/SUCCESS)
 Main PID: 666 (code=exited, status=1/FAILURE)

Mar 06 19:57:29 talisa systemd[1]: Starting Grafana Service Daemon...
Mar 06 19:57:31 talisa systemd[1]: Started Grafana Service Daemon.
Mar 06 19:57:32 talisa grafana[666]: 2016/03/06 19:57:32 [log.go:75 Fatal()] [E] Failed to detect generated css or javascript files in static root (%!s(MISSING)), have you executed default grunt task?
Mar 06 19:57:32 talisa systemd[1]: grafana.service: Main process exited, code=exited, status=1/FAILURE
Mar 06 19:57:32 talisa systemd[1]: grafana.service: Unit entered failed state.
Mar 06 19:57:32 talisa systemd[1]: grafana.service: Failed with result 'exit-code'.
```

Does anyone know how to fix it?

**Comment by bjornfor** [comment]
Diffing the .service files for those two commits:

```
--- good.service	2016-03-06 20:52:04.938389068 +0100
+++ bad.service	2016-03-06 20:52:00.989553608 +0100
-Environment="GF_SERVER_STATIC_ROOT_PATH=/nix/store/51mb47ym16b8ipk56w3w26rs6wnamd9j-go1.5-grafana-v2.6.0-bin/share/grafana/public"
+Environment="GF_SERVER_STATIC_ROOT_PATH=/nix/store/dm30kl4i3kmp1gdmiapkl54rhi000a3y-go1.5-grafana-v2.6.0/share/grafana/public"
```

The path listed in bad.service doesn't exist, whereas the one in good.service does exist.

**Comment by bjornfor** [comment]
Ah, it's a side effect of 4fede53c0996938979d2966a69f108782a8c1c12!

**Comment by bjornfor** [comment]
I'll push a fix as soon as I've tested it on top of master.

**Comment by bjornfor** [comment]
Fixed in master (d99033beb928edb040e9302904b7a736344e5bbf) and release-16.03 (1db66c9f47c7694c43675dc62f18f7e3395b721c). (Release-15.09 unaffected, AFAIK.)
